tests/integration/query/simple/with_filter/with_not_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
```

Comment:
todo: I think a test for an empty `_not` clause is required.

Reply:
will do :)

Reply:
✅